lstm/bmi_lstm.py

```diff
@@ -59,6 +60,7 @@
 import pandas as pd
 import torch
 import yaml
+from yaml import CLoader
```

Comment:
Its not strictly required to have `pyyaml` with `libyaml` support. Lets guard the import so we can fallback.

```suggestion
try:
    from yaml import CSafeLoader as SafeLoader
except ImportError:
    from yaml import SafeLoader
```